catalog/spatial/geocoding/spatial-geocoding-localsearch/src/main/java/org/codice/ddf/spatial/geocoder/geonames/GeoNamesLocalIndex.java

```diff
@@ -82,4 +87,31 @@ public NearbyLocation getNearbyCity(String location) throws GeoEntryQueryExcepti
 
         return null;
     }
+
+    @Override
+    public Optional<String> getCountryCode(String locationWKT, int radius) {
+        try {
+            Optional<String> alpha2CountryCode = geoEntryQueryable.getCountryCode(locationWKT,
+                    radius);
+
+            if (alpha2CountryCode.isPresent()) {
+                try {
+                    String alpha3CountryCode =
+                            ISOFormatConverter.convert(ISOFormatConverter.ENGLISH_LANG,
+                                    alpha2CountryCode.get());
+                    return Optional.of(alpha3CountryCode);
+                } catch (MissingResourceException e) {
+                    LOGGER.debug(
+                            "Failed to convert country code {} to alpha-3 format. Returning empty value",
+                            alpha2CountryCode.get());
```

Comment:
I think so. The `ParseException` is concerned with getting the alpha-2 country code from a WKT, while the `MissingResourceException` is concerned with converting the alpha-2 country code to alpha-3 format. Both of these debugs should not occur at the same time.